docs_app/src/styles/2-modules/_label.scss

```diff
@@ -56,6 +56,10 @@ label.raised, .api-header label {
             &.impure-pipe {
                 background-color: $brightred;
             }
+
+            &.operator {
+                background-color: $brightred;
```

Comment:
I reused the same color as in `impure-pipe` class - there are no impure pipes in RxJS 🙂